_guides/analytics/analytics_basics.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: page
+title: Analytics&#58; the Basics
+order: 0
+---
+
+Start here to learn the basics about AMP analytics.
+
+{% include toc.html %}
+
+## Use amp-pixel or amp-analytics?
+
+AMP provides two components to meet your analytics and measurement needs:
+[amp-pixel](/docs/reference/amp-pixel.html) and
+[amp-analytics](/docs/reference/extended/amp-analytics.html).
+Both options send analytics data to a defined endpoint.
+
+If you are looking for behavior like a simple
+[tracking pixel](https://en.wikipedia.org/wiki/Web_beacon#Implementation),
+the `amp-pixel` component provides basic page view tracking;
+page view data gets sent to a defined URL.
+Some integrations with a vendor may call for this component,
+in which case they will specify the exact URL endpoint. 
+
+For most analytics solutions, use `amp-analytics`.
+Page view tracking works in `amp-analytics` too.
+But you can also track user engagement with any type of page content,
+including clicks on links, buttons, videos.
+And you can measure how far on the page the user scrolled,
+whether or not the user engaged with social media, and more.
+[Deep Dive into Amp Analytics](/docs/guides/analytics/deep_dive_analytics.html).
+
+As part of integrating with the AMP platform,
+many analytics providers have simplified `amp-analytics` configurations
+so that's it's easy to capture data and push to their tracking tools.
+Access vendor documentation from the
+[amp-analytics specification](/docs/reference/extended/amp-analytics.html).
+
+**Note:** You can use both `amp-pixel` and `amp-analytics` in your pages:
+`amp-pixel` for simple page view tracking,
+and `amp-analytics` for everything else.
+Keep in mind that simpler AMP pages are better for users,
+so if you don’t need the extra tags, don’t use them.
+
+## Create a simple analytics configuration
+
+Learn how to create a simple
+[amp-pixel](/docs/reference/amp-pixel.html) and
+[amp-analytics](/docs/reference/extended/amp-analytics.html) configuration.
+
+### Simple amp-pixel configuration
+
+To create a simple `amp-pixel` configuration,
+insert something like the following into the body of your AMP page:
+
+{% highlight html linenos %}
+<amp-pixel src="https://foo.com/pixel?RANDOM"></amp-pixel>
+{% endhighlight %}
+
+In this example,
+the page view data gets sent to the defined URL, along with a random number.
+The `RANDOM` variable is one of many
+[supported variables in the AMP platform](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md).
+Learn more about
+[Variable substitution](/docs/guides/analytics/analytics_basics.html#variable-substitution) here.
+
+The [amp-pixel](/docs/reference/amp-pixel.html)
+component is built-in,
+so you won’t need an inclusion declaration.
```

Comment:
inclusion declaration like you do for AMP's extended components, including `amp-analytics`.